### ISSUE TITLE
Fails at runtime with `@ffmpeg-installer/ffmpeg` package

### ISSUE THREAD

**Issue by styfle** [zeit/ncc#376]
## Steps to reproduce

```
yarn init -y
yarn add @ffmpeg-installer/ffmpeg @zeit/ncc@0.18.3 
echo "let {path} = require('@ffmpeg-installer/ffmpeg'); console.log(path);" > index.js
./node_modules/.bin/ncc build index.js
node dist
```

### Error Message

```
/Users/styfle/Code/foo/ffmpeg-installer-example/dist/index.js:85
    throw 'Could not find ffmpeg executable, tried "' + npm3Binary + '" and "' + npm2Binary + '"';
    ^
Could not find ffmpeg executable, tried "/Users/styfle/Code/ffmpeg-installer-example/dist/darwin-x64/ffmpeg" and "/Users/styfle/Code/ffmpeg-installer-example/dist/ffmpeg/node_modules/@ffmpeg-installer/darwin-x64/ffmpeg"
```

A similar message during deployment:

![image](https://user-images.githubusercontent.com/229881/57331835-8903e180-70e7-11e9-806f-080ba1861175.png)

### Notes

This package uses `__dirname` in the require() so it looks like it should work.

https://github.com/kribblo/node-ffmpeg-installer/blob/bbe5c71fce98e0450e919ba985c6560b79225039/index.js#L37

**Comment by guybedford** [comment]
It looks like this will need to be handled as a special case unfortunately.

**Comment by rauchg** [comment]
@guybedford I think that's acceptable, as long as we keep the special cases in a segregated place, and try to minimize them. That technique has served us well for `pkg`

**Comment by styfle** [comment]
Fixed in #378
